bridge/client/app/_components/ktb-edit-service/ktb-edit-service.component.ts

```diff
@@ -40,6 +46,73 @@ export class KtbEditServiceComponent implements OnDestroy {
       this.eventService.deletionProgressEvent.next({isInProgress: true});
       this.deleteService();
     });
+
+    this.project$?.pipe(
+      takeUntil(this.unsubscribe$),
+    ).subscribe(project => {
+      if (!this.fileTree && project && (project.gitRemoteURI || project.gitRemoteURI !== '') && this.serviceName) {
```

Comment:
Is the remoteURI check wrong? It enters the `if`, if it is `undefined`. I guess you meant `&&` instead of `||` but even then `project.gitRemoteURI !== ''` is not needed.
`if(project.gitRemoteURI)` is false if it is empty string

Reply:
Ok I was not sure about that and added it just in case. I will remove it.